Adds Squallwalker suit

swankcookie opened the pull request:
:cl: Swankcookie
rscadd: Squall Walker uniforms from the popular movie franchise, "spacefights"
/:cl:

![suit and helmet](https://cloud.githubusercontent.com/assets/15881946/12667785/89d1a136-c615-11e5-97b6-d68a1f0e1522.png)
EDIT: added on-persons
http://puu.sh/mOkBI/0b86c095d9.png

Commit:
Cease-and-Desist incoming...

Signed-off-by: swankcookie <__X__>

Commit:
Copyright avoidance measures: taken

Signed-off-by: swankcookie <__X__>

KazeEspada commented on code/modules/clothing/head/misc.dm:
```diff
@@ -71,6 +71,15 @@
 	flags_cover = HEADCOVERSEYES
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE
 
+/obj/item/clothing/head/stormtrooper
```
Shouldn't it be a child of helmets or space helmets?

swankcookie replied:
I was thinking it would be similar to the carborg helmet

KazeEspada commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -165,6 +165,14 @@
 	body_parts_covered = CHEST|GROIN
 	flags_inv = HIDEJUMPSUIT
 
+/obj/item/clothing/suit/stormtrooper
```
Space suits or armor?

swankcookie replied:
They are intended to be a cosmetic.

neersighted commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -165,6 +165,14 @@
 
+/obj/item/clothing/suit/stormtrooper
+	name = "squallwalker suit"
+	desc = "Cease-and-Desist incoming"
```
no kidding, change the icon_states

swankcookie replied:
wait is that an actual concern? jesus.

Commit:
fuck Disney™

Signed-off-by: swankcookie <__X__>

optimumtact commented:
:+1: TR8-T0r

PKPenguin321 commented:
top meme

swankcookie commented:
dont wanna

Boredone commented:
I feel like Squall Marcher would fit a bit better, but seems fine otherwise personally.

swankcookie commented:
well if you had even bothered to watch "spacefights 2: the government comes back" then you would know that they lack even the basic discipline to be called marchers.

Alek2ander commented:
Bug: does not make you miss with a laser gun

xxalpha commented:
>Bug: does not make you miss with a laser gun

This.

Zilenan91 commented:
Is this gonna be admin spawn only or a costume in the AutoDrobe?

swankcookie commented:
As of right now im just putting the costumes in for admin spawn, but I could add them to drobes if i see positive feedback

Zilenan91 commented:
I think it'd fit in nicely if you tossed it in the drobe

Edit : Oh could you also toss the xeno suit in there too?

swankcookie commented:
yes and yes

swankcookie commented:
EDIT: added on-persons

Commit:
moved vending_items.dm out of weapons folder
added squallwalker to Drobe
added Xenos to contraband Drobe
deleted 2 unused sprites (shitty unused new years tree)

Signed-off-by: swankcookie <__X__>

Wyzack commented:
You fucking newfag that was spacefights 5

freerealestate commented:
oh god please no

swankcookie commented:
I thought it all went downhill after "the ghostly annoyance"

optimumtact commented:
Honestly it really got bad when they introduced Jam Jar Bings

tkdrg commented on code/game/machinery/vending.dm:
```diff
@@ -870,8 +870,10 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
+					/obj/item/clothing/head/snowman = 1,
+					/obj/item/clothing/suit/squallwalker = 1,
+					/obj/item/clothing/head/squallwalker = 1)
+	contraband = list(/obj/item/clothing/suit/judgerobe = 1,/obj/item/clothing/head/powdered_wig = 1,/obj/item/weapon/gun/magic/wand = 2,/obj/item/clothing/glasses/sunglasses/garb = 2,/obj/item/clothing/suit/xenos,/obj/item/clothing/head/xenos)
```
> /obj/item/clothing/suit/xenos,/obj/item/clothing/head/xenos)

????
also the format here is broken

tkdrg commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -165,6 +165,14 @@
 
+/obj/item/clothing/suit/squallwalker
+	name = "squallwalker suit"
+	desc = "Cease-and-Desist incoming"
```
awful meme desc, also wrong capitalization and lacks punctuation

swankcookie replied:
3/4ths of the descriptions in ss13 are memes, but fine.

tkdrg commented on code/game/objects/items/vending_items.dm:
```diff
@@ -1,73 +1,73 @@
-/obj/item/weapon/vending_refill
```
Why are you renaming this file?

swankcookie replied:
I dint. i moved it out of weapons because it made no sense being in there

tkdrg commented on code/game/objects/items/vending_items.dm:
```diff
@@ -1,73 +1,73 @@
+/obj/item/weapon/vending_refill/autodrobe
+	machine_name = "AutoDrobe"
+	icon_state = "refill_costume"
+	charges = list(29, 4, 3)// of 75 standard, 6 contraband, 9 premium
```
1) update the comment, it's now 77 standard 8 contraband 9 premium, assuming the previous comment was accurate
2) the count is wrong, these are meant to be 1/3 of total rounded up, so it should be 26 3 3

optimumtact replied:
delete the comment

tkdrg replied:
i dunno, i think those are useful as reminders of the right ratio for charges

swankcookie replied:
The ratios were already loose as fuck anyway

tkdrg commented:
Also, why are you editing icons/effects/224x224.dmi and  icons/effects/160x160.dmi ?

Nienhaus commented:
https://www.dropbox.com/s/kw761uqfzce3wxv/TR-8R%20weapon.dmi?dl=0

Alek2ander commented:
These are not the memes you're looking for.

swankcookie commented:
@tkdrg I deleted 2 completely unused, multiple-year-old icons

Commit:
general fixes

Signed-off-by: swankcookie <__X__>

Commit:
this too.

Zilenan91 commented:
is this ready to merge?

KorPhaeron commented:
@swankcookie 

The reality tear is not unused, the veil render uses it.

You shouldn't be removing stuff without telling anyone either

swankcookie commented:
@KorPhaeron its in my commit logs. although i guess i should know that nobody reads those.

Commit:
accidentally deleted something

Signed-off-by: swankcookie <__X__>

swankcookie commented:
@whoeverwantstomergethis Ready

The-Albinobigfoot commented:
Preferably this wouldn't make it into the game at all, but since everyone seems to be eating it up, all I ask is that you keep it as admin spawn only. Same for the Xeno suit, or any other "gimmicky" admin item.

Zilenan91 commented:
Xeno suits aren't an admin item, you can make them from dead xenos and they're on Metastation.

The-Albinobigfoot commented:
>you can make them from dead xenos

Isn't that more of a reason to keep them out of the autodrobe, then?

>Metastation

Frankly, it shouldn't be there to begin with.

>putting cosmetic costumes as admin-only is stupid

Not always, it allows the admins to run a (potentially) interesting gimmick with players, rather than just hurling meteors or death squads at us. (e.g. Chrono Legionnaires [Yeah, I know it's not entirely cosmetic but still]).

swankcookie commented:
So due too raisons,I can no longer work on my repo, so I cant fix the conflicts. If someone wants this to go through, then they are welcome to take the sprites and remake the PR. (just credit me plox)

tkdrg commented:
If this particular PR won't go anywhere, I'll close it, then.

optimumtact commented:
awww

swankcookie commented:
@tkdrg  oh yeah, good call

swankcookie commented:
sorry, oranges. your welcome to just steal the sprites and redo the PR, though
